# Define the goal

uint8  START = 1
uint8  STOP = 2
uint8  SUSPEND = 3
uint8  CONFIRM_STOP=4

uint8  VERTEX_TYPE=100
uint8  CURVE_TYPE =101

uint8  command_type
uint8  path_type
string path_file_name
float32 speed

---
# Define the result

uint8 SUCCESS = 0
uint8 PATH_FILE_NOT_EXIST = 1
uint8 FAIL = 2
uint8 PATH_FILE_ERROR = 3
uint8 COMPLETE = 4

uint8 result

---
# Define the feedback
float32 percent_complete
float32 remaind_distance



